ui/src/app/settings/components/Nav/Nav.js

```diff
@@ -0,0 +1,66 @@
+import { HashLink } from "react-router-hash-link";
+import { Link } from "react-router-dom";
+import { withRouter } from "react-router";
+import classNames from "classnames";
+import React from "react";
+import ReactRouterPropTypes from "react-router-prop-types";
+
+import "./Nav.scss";
+
+function _generateSection(section, location) {
```

Comment:
We don't tend to use the ES5 `function` keyword anymore. There's nothing wrong with it, but for consistency, can we use ES6 arrow functions exclusively?

Reply:
Done.